src/main/java/org/openelisglobal/dictionary/controller/DictionaryMenuController.java

```diff
@@ -1,185 +1,40 @@
 package org.openelisglobal.dictionary.controller;
 
-import java.lang.reflect.InvocationTargetException;
```

Comment:
@mherman22 , 
you definately can ot remove the old JSP controllers. 
Create a new subfolder `controller/rest`  and write a new REST controller for the new UI. 
We need to still keep the old UI working

Reply:
oh cool, will do that. Thanks for chipping in before i went deep into disorganizing things 👍🏿